plugins/mysql/mysql-host-errors.rb

```diff
@@ -0,0 +1,113 @@
+#! /usr/bin/env ruby
+#
+#   <script name>
+#
+# DESCRIPTION:
+#   what is this thing supposed to do, monitor?  How do alerts or
```

Comment:
Can you fill th header out here so it's clear what the plugin does and how it should be used?